src/_guides/language/sound-dart.md

```diff
@@ -183,35 +145,26 @@ A sound type system has several benefits:
   code is much less efficient.
 
 * Cleaner JavaScript.<br>
-  For web apps, strong mode's more restrictive typing allows
-  dartdevc to generate cleaner, more compact JavaScript.
-
-## What constitutes strong mode?
-
-Dart's strong mode implementation, which enables soundness, consists
-of three pieces:
+  For web apps, sound typing allows
+  [dartdevc][] to generate cleaner, more compact JavaScript.
```

Comment:
Should we combine "Better AOT" with "Cleaner JS" (essentially the latter is a special case of the former).

Reply:
I think I'll just delete the "cleaner JS" unless someone objects.